eventhubs/client/azure-eventhubs/src/main/java/com/azure/core/amqp/AmqpExceptionHandler.java

```diff
@@ -18,7 +18,7 @@
      * Creates a new instance of an exception handler.
      */
     protected AmqpExceptionHandler() {
-        this.logger = new ServiceLogger(AmqpExceptionHandler.class);
+        this(new ServiceLogger(AmqpExceptionHandler.class));
```

Comment:
There is an issue here - ServiceLogger is currently in the implementation package, meaning it shouldn't be in public API. We should perhaps chat with @alzimmermsft to move this into public API.

Reply:
Ahh. Fair enough. I followed the example from ApplicationConfig. I agree we should move this out. 👍